docs/content/tools/_index.md

```diff
@@ -13,20 +13,47 @@ toc:
 
 The [OSCAL models](/concepts/layer/) provide standardized formats for exchanging control, control implementation, and control assessment information in XML, JSON, and YAML. These formats allow this information to be exchanged between tools and for individual tools to process exchanged data, supporting analytics, user interaction, and increased automation.
 
-Tools exist that support the use of the OSCAL models. These tools are listed below in the following categories:
+Tools supporting the use of the OSCAL models are listed below in the following categories:
 
-- OSCAL Project provided tools and libraries
-- Open Source Tools provided by 3rd parties
+- Commodity tooling supporting basic operations
```

Comment:
'basic operations' is referring to editorial operations? Maybe we should group the tool as: 
- OSCAL content editorial tools (including format conversion and data visualization), 
- OSCAL governance, risk, and compliance (GRC) tools and applications

Anything more consistent in terms of listing criteria is what I am looking for. It should be either by the authorship (NIST vs non-NIST) or by the functionality.

Reply:
No, the distinction is between 'basic operations' such as validating against a public schema, converting data, or transforming using generic tooling.

This reflects the distinction is between "things you will find in the OSCAL repository under /json or /xml" and "other things".

@iMichaela, this page revision is happening because people were (are) coming to the Tools page and not seeing that we have validators and converters available for public use - they were not linked or even mentioned.